doc/tutorials/acrn_on_qemu.rst

```diff
@@ -0,0 +1,288 @@
+.. _acrn_on_qemu:
+
+Enable ACRN over QEMU/KVM
+#########################
+
+Goal of this document is to bring-up ACRN as a nested Hypervisor on top of QEMU/KVM
+with basic functionality such as running Service VM (SOS) and User VM (UOS) for primarily 2 reasons,
+
+1. In order for the users to evaluate ACRN.
+2. Make ACRN platform agnostic and remove per hardware platform configurations setup overhead.
+
+This setup was tested with the following configuration,
+
+1. ACRN Hypervisor: tag acrn-2020w19.5-140000p (Commit ID: b0ae9cfa2b374d3faad9f890ff61e88f97205ac8)
+2. ACRN Kernel: 5.4.28 (Commit ID: a8cd22f49f0b2b56e526150fe0aaa9118edfcede)
+3. QEMU emulator version 4.2.0
+4. Service VM/User VM is ubuntu18.04
+5. Platforms Tested: ApolloLake, KabyLake, CoffeeLake
+
+
+Pre-Requisites
+**************
+1. Make sure the platform supports Intel VMX as well as VT-d technologies. On ubuntu18.04.4, this
+   can be checked by installing ``cpu-checker`` tool. If the output displays **KVM acceleration can be used**
+   the platform supports it.
+
+  .. code-block:: none
+
+     acrn@host:~$ kvm-ok
+     INFO: /dev/kvm exists
+     KVM acceleration can be used
+
+2. Make sure KVM and the following utilities are installed.
+
+  .. code-block:: none
+
+     sudo apt update && sudo apt upgrade -y
+     sudo apt install qemu-kvm libvirt-bin virtinst -y
+
+
+Prepare Service VM (L1 Guest)
+*****************************
+1. Use ``virt-install`` command to create Service VM.
+
+   .. code-block:: none
+
+      virt-install \
+      --connect qemu:///system \
+      --name ACRNSOS \
+      --machine q35 \
+      --cpu host-passthrough,+invtsc \
+      --ram 4096 \
+      --disk path=/var/lib/libvirt/images/acrnsos.img,size=32 \
+      --vcpus 4 \
+      --virt-type kvm \
+      --os-type linux \
+      --os-variant ubuntu18.04 \
+      --graphics none \
+      --clock offset=utc,tsc_present=yes,kvmclock_present=no \
+      --qemu-commandline="-machine kernel-irqchip=split -device intel-iommu,intremap=on,caching-mode=on,aw-bits=48" \
+      --location 'http://archive.ubuntu.com/ubuntu/dists/bionic/main/installer-amd64/' \
+      --extra-args "console=tty0 console=ttyS0,115200n8"
+
+2. Walk through the installation steps as prompted. Few things to note,
+
+   a. Make sure to install an OpenSSH server so that once the installation is complete, we can SSH into the system.
+
+      .. figure:: images/acrn_qemu_1.png
+         :align: center
+
+   b. We use GRUB to boot ACRN, so please install it when prompted.
+
+      .. figure:: images/acrn_qemu_2.png
+         :align: center
+
+3. To login to the Service VM guest, find the IP address of the guest to SSH. This can be done via the
+   virsh command as shown below,
+
+   .. figure:: images/acrn_qemu_3.png
+      :align: center
+
+4. Once ACRN hypervisor is enabled, the above virsh command might not display the IP. So enable Serial console by,
```

Comment:
This tip should probably be before the previous one (showing the virsh command).  If you leave it here, you should also say to run the virsh command again after enabling the serial console.

Reply:
This command need not be run after enabling the serial console. The point I am trying to make here is that virsh commands like this will not work once ACRN hypervisor is enabled. So we need to have console to display grub options.